vmdb/app/models/host.rb

```diff
@@ -1400,6 +1399,14 @@ def connect_ssh(options={})
       rl_user, rl_password = self.auth_user_pwd(:root)
       su_user, su_password = nil, nil
     end
+    return rl_user, rl_password, su_user, su_password, {}
+  end
+
+  def connect_ssh(options={})
+    require 'MiqSshUtil'
+
+    rl_user, rl_password, su_user, su_password, additional_options = ssh_users_and_passwords
```

Comment:
@jrafanie rebased as you wanted in the followup PR. Looks much nicer, thank you.